scylla-cql/src/types/serialize/value.rs

```diff
@@ -2069,6 +2069,44 @@ mod tests {
         assert_eq!(reference, udt);
     }
 
+    #[test]
```

Comment:
> This commit adds tests for new behavior of SerializeCql - ignoring
unknown UDT fields.\

Nit: second commit message: unnecessary backslash at the end of the line